sidebars.json

```diff
@@ -1,7 +1,7 @@
 {
   "docs": {
     "Introduction": ["introduction/what-is-tradetrust", "introduction/tradetrust-layer-boundaries"],
-    "Overview": ["overview/issuer-dns-txt-record", "overview/token-registry", "overview/title-transfer"],
+    "Overview": ["overview/issuer-dns-txt-record", "overview/token-registry", "overview/title-transfer", "overview/title-transfer-example"],
     "Technical": ["technical/open-attestation"],
```

Comment:
could nest this 
```suggestion
    "Overview": ["overview/issuer-dns-txt-record", "overview/token-registry", "overview/title-transfer", {
        "label": "Example",
        "type": "category",
        "items": [
          "overview/title-transfer/example",
        ]
      }"],
    "Technical": ["technical/open-attestation"],
```

i think

Reply:
k

Reply:
oops i meant sth like this: 

<img width="723" alt="Screenshot 2021-09-02 at 4 11 00 PM" src="https://user-images.githubusercontent.com/4774314/131807485-57e8b230-fb97-47b2-97c2-df89f849b240.png">

overview = the existing copy of title transfer background
title transfer example = the new docs in this pr

then can have subsequent `surrender example` topic later on

Reply:
👌